docs/troubleshooting/README.md

```diff
@@ -0,0 +1,15 @@
+---
+order: 9
+---
+
+# Troubleshooting
+
+## Memory error
+
+The default amount of memory available for Docker on macOS is often insufficient to get Kestra up and running with all the dependencies. If enough memory is not allocated, it might lead to the Kestra instance continuously restarting. It would be best to allocate at least 4 GB memory for the Docker Engine (ideally 8 GB). You can check and change the amount of memory in [Resources](https://docs.docker.com/docker-for-mac/#resources).
```

Comment:
I'm surprised we need this much memory for the Docker VM. Can someone check if it's still needed with the default JDBC plug-ins?

Reply:
@tchiotludo Thoughts?

Reply:
we don't need anymore such amount, 2Go will be sufficient now, it was an history based on kaafka + elastic that need approximately 4go

Reply:
Should I get rid of the Troubleshooting page + that note, then?

Reply:
you are on mac with default config with no issue ? if yes, you can

Reply:
That's correct. It ran smoother than butter with the default config.

Reply:
Nice catch @loicmathieu 🙌